packages/ui/src/components/templates/report/adapters/report-adapter.ts

```diff
@@ -23,6 +22,47 @@ export const toRiskLabels = (riskIndicators: Array<{ name: string; riskLevel: st
   }));
 };
 
+export const toSocialMediaPresence = (data: Record<string, any>) => {
```

Comment:
WDYT about using `unknown` instead of `any`? I think we should try to use `any` less often because it weakens type safety, which can cause runtime errors and make the code harder to maintain.

Reply:
sgtm, i'd say that the best approach would be to type the responses/normalized data in the end, so any temporary loose type here works and unknown is prolly better indeed.

Reply:
<img width="674" alt="image" src="https://github.com/user-attachments/assets/6a82681c-4866-4d78-acd6-9217bd6c91d2" />

however, using unknown basically makes it a bit more strict and sort of forces dev to type it properly. unsure if we want to prioritize that right now since from my observations data coming from agents gets transformed a lot in the middle, i'd start with simplifying the pipelines that the data goes through first tbh along with properly typing incoming data. we have quite a lot of anys in the codebase anyway. it might be a good candidate for a tech debt epic.